src/protobuf.py

```diff
@@ -204,8 +207,9 @@ def __init__(self):
         setattr(msg, fname, fvalue)
 
     # fill missing fields
-    for tag in msg.FIELDS:
-        field = msg.FIELDS[tag]
+    fields = msg.get_fields()
```

Comment:
Redundant?

Reply:
Nonono, I mean that `fields` already contains the message fields, since line 155.

Reply:
😅 true